sirepo/job_supervisor.py

```diff
@@ -431,6 +432,7 @@ def __db_load(cls, compute_jid):
                 'isPremiumUser',
                 'jobStatusMessage',
                 'internalError',
+                # TODO(e-carlin): dbUpdateTime?
```

Comment:
@robnagler what's the proper way to do a fixup for this. Just use srtime.utc_now_as_float()? Try and look back in the history and use another time (ex lastUpdateTime)?

Reply:
`__db_write` should set `dbUpdateTime` to now before writing. This should cover all cases except [this one](https://github.com/radiasoft/sirepo/blob/6d285a3f4e33e9f4871aa63417f5451f8947d4a0/sirepo/job_supervisor.py#L839), which seems to be a bug. There's a db_init when the run_dir is not found, but there's no db_update/write.

Reply:
I'll make those changes. But, we still have a problem. If a runStatus is the first request a __db_write() is never called so dbUpdateTime won't be on the db object.

Reply:
I was making the point that [this case](https://github.com/radiasoft/sirepo/blob/6d285a3f4e33e9f4871aa63417f5451f8947d4a0/sirepo/job_supervisor.py#L839) needs to call db_write, because it is creating a db object, returning it, and not writing it.

Reply:
Ok, I'll fix that. 

Can you answer my original question:

> robnagler what's the proper way to do a fixup for this. Just use srtime.utc_now_as_float()? Try and look back in the history and use another time (ex lastUpdateTime)?

We need to add dbUpdateTime for all of the jobs in the db that don't have it so we can display it in the GUI.

Reply:
https://github.com/radiasoft/sirepo/pull/2690/commits/c8a7613fdb8c6f1d53c4cc8e1569628943fcd152 fix all time formats.

https://github.com/radiasoft/sirepo/pull/2690/commits/c8a7613fdb8c6f1d53c4cc8e1569628943fcd152 fix missing db_write